src/applications/facility-locator/utils/facilityAddress.js

```diff
@@ -27,7 +27,11 @@ export function buildAddressArray(location, titleCaseText = false) {
 
     return [];
   }
-  if (location && location.attributes) {
+  if (
+    location &&
+    location.attributes &&
+    !Array.isArray(location.attributes.address)
+  ) {
```

Comment:
I think we may be better off verifying we have the necessary data to create the address.
We could shorten this down to 
```suggestion
 if (location?.attributes?.address?.physical) {
```
using optional chaining